cetmix_tower_server/models/__init__.py

```
@@ -14,6 +14,7 @@
 from . import cx_tower_os
 from . import cx_tower_tag
 from . import cx_tower_command
+from . import cx_tower_custom_variable_value_mixin
```

Flake8 (7.2.0)

[error] '.cx_tower_custom_variable_value_mixin' imported but unused

(F401)